app/public/components/userView/data/table/filter.js

```diff
@@ -81,25 +81,27 @@ const createClickableLabel = (model, field) =>
             style: 'width:120px',
             onclick: () => model.fetchedData.changeItemStatus(field),
             className: field.marked ? 'active' : '',
-        }, headersSpecials[model.getCurrentDataPointer().page][field.name],
+        }, getHeaderSpecial(model, field),
         h('span.tooltiptext', field.marked ? 'hide' : 'display')));
 
-const createInputField = (inputId, currentValue) =>
-    h('th.my-tooltip.noBorderBottom.table-cell-like',
+const createInputField = (inputId, currentValue) => {
+    const type = inputId.substring(inputId.indexOf('-') + 1);
+    return h('th.my-tooltip.noBorderBottom.table-cell-like',
         h('div.rel',
             h('input.form-control', {
                 style: 'width:120px',
                 type: 'text',
                 value: currentValue ? currentValue : '',
+                disabled: type == null,
                 id: inputId,
             }),
-            h('span.tooltiptext', `${inputId.substring(inputId.indexOf('-') + 1)}`)));
-
+            h('span.tooltiptext', `${type}`)));
+};
 const onclickSubmit = (model, inputsIds) => () => {
     const filteringParamsPhrase = inputsIds
         .map((inputId) => [
             inputId,
-            [...document.getElementById(inputId)?.value || ''].map((c) => c == '%' ? '%25' : c).join(''),
+            encodeURI(document.getElementById(inputId)?.value || ''),
```

Comment:
Oh, how could I forget about this function back when writing this code for the first time :smile_cat: